tests/framework/ab_test.py

```diff
@@ -82,19 +82,20 @@ def git_ab_test(
              (alternatively, your comparator can perform any required assertions and not return anything).
     """
 
-    dir_a = git_clone(Path("../build") / a_revision, a_revision)
-    result_a = test_runner(dir_a, True)
+    with TemporaryDirectory() as tmp_dir:
```

Comment:
I don't understand why do they clash? What is the use context here? If we are running in Buildkite, we will have the 2 directories set up for us, and if they don't then there's nothing to clash against?

Reply:
The problem here was that buildkite doesn't provide the entire workspace anymore, but rather just the binaries, but this script still thought it was the entire workspace.